### DISCUSSION TITLE
Implement Force Start/Resume and Reannounce.

### DISCUSSION THREAD

**Discussion by lilws** [jesec/flood#191] [Q&A] [answered]
I wonder if these 2 features can be implement in flood?
![Picture1](https://user-images.githubusercontent.com/12478478/107121606-ebde6a00-68c5-11eb-8109-f34e2202b32c.png)

**Comment by jesec** [comment, marked as answer]
`Force Resume` is related to queue. It does not make sense without general queueing support. 

`Reannounce` is a good idea.

For reference: 

rTorrent: https://rtorrent-docs.readthedocs.io/en/latest/cmd-ref.html#term-d-tracker-announce
qBittorrent: https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-(qBittorrent-4.1)#reannounce-torrents
Transmission: https://github.com/transmission/transmission/blob/090a4b5ff8f6b31a4eea4bdb75eb1350aa3efae0/extras/rpc-spec.txt#L94

**Reply by lilws** [reply]
In rtorrent there is no queue so it could be off-topic, but since qBittorrent and Transmission do have queue, sometimes in flood I want override that settings for a session, Implement queue might take a lot of time, but force resume I think it could be possible. It's just a request of UI, I think. But it's all up to you 👍

**Reply by lilws** [reply]
Ah and one more thing, when in qbittorrent I force resume a torrent (downloading). In flood, it's not appear in any filter status like downloading or active. Its status like stop torrent and green for downloading like this picture and not in any filter status except All.
![Picture1](https://user-images.githubusercontent.com/12478478/107123170-a5413d80-68ce-11eb-80b6-fbd0ce0b704b.png)

**Reply by jesec** [reply]
Apparently this state is not handled. I will take a look.

**Reply by jesec** [reply]
resolved by https://github.com/jesec/flood/commit/20ed7f12adf2bfbc93302aa115349c3cbe6266f3.
